packages/client/modules/userDashboard/containers/UserColumns/UserColumnsContainer.tsx

```diff
@@ -85,7 +85,7 @@ const UserColumnsContainer = (props: Props) => {
       <TaskColumns
         area={areaForTaskCard}
         tasks={filteredTasks}
-        myTeamMemberId={toTeamMemberId(filteredTeams[0].id, userIds ? userIds[0] : viewer.id)}
+        myTeamMemberId={toTeamMemberId(filteredTeams[0]!.id, userIds ? userIds[0]! : viewer.id)}
```

Comment:
Would be nicer if you would replace the `if(filteredTeams.length)` with
```
const firstTeam = filteredTeams[0]
if (firstTeam) {
```
and then in this line with
```
        myTeamMemberId={toTeamMemberId(firstTeam.id, userIds?.[0] ?? viewer.id)}
```